armi/physics/executers.py

```diff
@@ -178,11 +178,18 @@ def run(self):
             )
         self._performGeometryTransformations()
         inputs, outputs = self._collectInputsAndOutputs()
+        state = f"c{self.r.p.cycle}n{self.r.p.timeNode}"
+        outputDir = os.path.join(
+            pathTools.armiAbsPath(os.getcwd()), state, self.options.label
+        )
```

Comment:
This change requires a change to our `.gitignore` file as well, right?

Because this change would definitely pollute the repo with all kinds of extra folders?